src/main/java/com/revature/assignforce/validators/IsValidIntervalValidator.java

```diff
@@ -0,0 +1,93 @@
+package com.revature.assignforce.validators;
+
+import java.lang.reflect.Field;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+/**
+ *
```

Comment:
Same here. Add doc comments. Also, please list your full name rather than just Hayden. This project is worked on by hundreds of people, so we others won't know who Hayden is.